tests/src/test/scala/scalaz/ValidationTest.scala

```diff
@@ -77,12 +78,12 @@ object ValidationTest extends SpecLite {
     "return first success" in {
       val succ = Validation.success[String, Int](1).toValidationNel
       val fail = Validation.failure[String, Int]("2").toValidationNel
-    
```

Comment:
I somewhat want to go through scalaz (using something automated like codemod) and fix up all random trailing whitespace like this. Would that be annoying? It would mess up git history slightly, but it is annoying in editors/git diffs which show it in painfully bright red like my emacs setup does. Thoughts?